acceptance/acceptance_test.go

```diff
@@ -15,40 +14,111 @@ import (
 	h "github.com/buildpacks/lifecycle/testhelpers"
 )
 
+var dir string
+
 func TestAcceptance(t *testing.T) {
+	if os.Getenv("ACCEPTANCE") == "" {
```

Comment:
Why not just use the `+build acceptance` annotation?

Reply:
How do you then execute this via IDE do you need to configure a run configuration with the env var? That seems more painful IMO.

We haven't had any issues with build constraints in `pack`. Could you elaborate?

Reply:
It's about the same mount of work either way. In Goland for example, you either need a Run Configuration for the acceptance tests with `ACCEPTANCE` set or you need to add a custom build tag to your Project Settings.

I would say that the advantages of the env var are:
* Goland will compile the file for you by default regardless (and show compilation errors)
* If you try to run it without the appropriate setup it explicitly prints out why it is skipping

Reply:
In my experience adding the build constraint is an auto-fix action with the IDE. Is that not the case for you?